mne/viz/_3d.py

```diff
@@ -1719,7 +1719,9 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
     else:
         hemis = [hemi]
 
-    if title is None:
+    if title is None and isinstance(time_label, str):
```

Comment:
Actually instead of using `time_label` you could just pass `title`. Sorry I forgot it was an argument at all. Then you don't need to add this conditional.

So for the code in the example you can do `title = 'MNE...'; stc.plot(time_label=title, title=title, ...)` and it will show up in the title bar and in the image itself. Is that what you want?

Reply:
Yes, that's perfect!